src/app/routes/index.js

```diff
@@ -1,16 +1,24 @@
-import { Route, Switch } from 'react-router-dom';
 import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import PropTypes from 'prop-types';
 import ChannelsListContainer from '../features/Chanels/containers/ChannelsListContainer';
 import ChannelPageContainer from '../features/Chanels/containers/ChannelPageContainer';
 import 'antd/dist/antd.css';
+import LogInPage from '../components/LologInPage';
 
-const RoutesResolver = () => {
-  return (
+const RoutesResolver = ({ authorized }) => {
+  return authorized ? (
```

Comment:
I've found a bug. If I log in once, and then clear localStorage, app refuses to log in next time. I get the following error: 

`Call to setAuthenticationPhoneNumber unexpected`

Please, check this behavior. 

I guess the solution would be to use telegram api to check if user is authenticated. So, to store the current user number in localStorage and call telegram api initially to check authorization status, rather than storing **authorized** flag in localStorage